packages/repository-json-schema/src/build-schema.ts

```diff
@@ -16,10 +16,44 @@ import {JSONSchema6 as JSONSchema} from 'json-schema';
 import {JSON_SCHEMA_KEY, MODEL_TYPE_KEYS} from './keys';
 
 export interface JsonSchemaOptions {
+  /**
+   * Set this flag if you want the schema to define navigational properties
+   * for model relations.
+   */
   includeRelations?: boolean;
+
+  /**
+   * Set this flag to mark all model properties as optional. This is typically
+   * used to describe request body of PATCH endpoints.
+   */
+  partial?: boolean;
```

Comment:
In the future, we can support more options such as `exclude`.

Reply:
Definitely, see e.g. https://github.com/strongloop/loopback-next/issues/2653:

> Allow Controller methods implementing CREATE operation to describe their input data as "a model without `id` and `_rev` properties ". This story requires #2629 and #2631 to be implemented first.
> 
> To emit schema with certain properties excluded, we should add a new schema-generation option called `exclude` & accepting a list of property names to remove from the schema. To allow TypeScript compiler to verify that `exclude` items are valid property names, we should leverage TypeScript generics and `keyof` keyword.